### PR TITLE
feat: Rename queryTransformer to queryRewrite (with compatibility)

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: docs/content/Configuration/Config.md [hunk 1]
```diff
@@ -262,20 +262,25 @@ module.exports = {
 };
 ```
 
-### queryTransformer
+### queryRewrite
+
+<!-- prettier-ignore-start -->
+[[warning | Note]]
+| In previous versions of Cube.js it was called `queryTransformer`.
```
Comment (hassankhan) [inline docs/content/Configuration/Config.md:269]: | In previous versions of Cube.js, this was called `queryTransformer`.

**Review by hassankhan — COMMENTED** [review]
Looks good on the surface to me; I'll pull the branch and check for any dead links 👍

**Review by hassankhan — APPROVED** [review]
No dead links found; LGTM 👍

File: docs/content/Configuration/Config.md [hunk 2]
```diff
@@ -262,20 +262,25 @@ module.exports = {
 };
 ```
 
-### queryTransformer
+### queryRewrite
+
+<!-- prettier-ignore-start -->
+[[warning | Note]]
+| In previous versions of Cube.js, this was called `queryTransformer`.
```
Comment (keydunov) [inline docs/content/Configuration/Config.md:269]: @ovr It is great that you have added this note, thank you! @hassankhan do you think we need to mention something around that `queryTransformer` name still supported and that we kept the backward compatibility?

**Pull request merged**